"set(ENABLE_SIMPLESTOT_ASM, OFF)" not work

Hi, I'm facing a compile problem when building my project using libOTe as a submodule. In CMakeLists.txt, I wrote:
``` cmake
set(ENABLE_RELIC ON CACHE BOOL "Build relic by default" FORCE)
set(MULTI PTHREAD CACHE STRING "Build relic with multithreading from pthread" FORCE)
set(ARITH easy CACHE STRING "Build relic without gmp" FORCE)
set(WITH "MD;DV;BN;FB;FP;EB;EC;EP" CACHE STRING "Overwrite Relic algorithms in ABY with their superset" FORCE)

# we need to set these compile flags globally to compile libOTe and its dependencies
set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -no-pie -pthread -maes -msse2 -msse3 -msse4.1 -mpclmul -mavx -mavx2")
set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-ignored-attributes")

find_package(libOTe QUIET)
if (libOTe_FOUND)
    message(STATUS "Found libOTe")
elseif (NOT libOTe_FOUND AND NOT TARGET libOTe::libote)
    message("libOTe was not found: add libOTe subdirectory")
    if (NOT EXISTS "${PROJECT_SOURCE_DIR}/extern/libOTe/CMakeLists.txt")
        find_package(Git REQUIRED)
        message("initialize Git submodule: extern/libOTe")
        execute_process(COMMAND git submodule update --init --recursive extern/libOTe
                WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}")
    endif ()
    add_subdirectory(extern/libOTe)
endif ()
set(ENABLE_SIMPLESTOT ON CACHE BOOL "Enable Simplest OT for Base OTs" FORCE)
set(ENABLE_IKNP ON CACHE BOOL "Enable IKNP for 1-out-of-2 OTs" FORCE)
set(ENABLE_KKRT ON CACHE BOOL "Enable KKRT for OPRFs" FORCE)
```

And indeed, output shows that `ENABLE_SIMPLESTOT_ASM` is OFF:
```
[cmake] -- Base OT protocols
[cmake] =======================================================
[cmake] -- Option: ENABLE_SIMPLESTOT     = ON
[cmake] -- Option: ENABLE_SIMPLESTOT_ASM = OFF
[cmake] -- Option: ENABLE_MR             = OFF
[cmake] -- Option: ENABLE_MR_KYBER       = OFF
[cmake] -- Option: ENABLE_NP             = OFF
[cmake] 
[cmake] 
[cmake] -- 1-out-of-2 OT Extension protocols
[cmake] =======================================================
[cmake] -- Option: ENABLE_KOS            = OFF
[cmake] -- Option: ENABLE_IKNP           = ON
[cmake] -- Option: ENABLE_SILENTOT       = OFF
[cmake] 
[cmake] 
[cmake] -- 1-out-of-2 Delta-OT Extension protocols
[cmake] =======================================================
[cmake] -- Option: ENABLE_DELTA_KOS      = OFF
[cmake] -- Option: ENABLE_DELTA_IKNP     = OFF
[cmake] 
[cmake] 
[cmake] -- 1-out-of-N OT Extension protocols
[cmake] =======================================================
[cmake] -- Option: ENABLE_OOS            = OFF
[cmake] -- Option: ENABLE_KKRT           = ON
[cmake] -- Option: ENABLE_RR             = OFF
```

However, when compiling my project, it reports the error:
```
undefined reference to `receiver_procS' in 'extern/libOTe/libOTe/Base/SimplestOT.cpp'
undefined reference to `sender_genS' in 'extern/libOTe/libOTe/Base/SimplestOT.cpp'
...
```
The reason should be `#ifdef ENABLE_SIMPLESTOT_ASM` is enabled in SimplestOT.cpp. Why would this happen? I've checked the content in file `Debug/extern/libOTe/libOTe/config.h` is actually
``` hpp
// build the library with "simplest" Base OT enabled
#define ENABLE_SIMPLESTOT ON

// build the library with the ASM "simplest" Base OT enabled
/* #undef ENABLE_SIMPLESTOT_ASM */

// build the library with Masney Rindal Base OT enabled
/* #undef ENABLE_MR */

// build the library with Masney Rindal Kyber Base OT enabled
/* #undef ENABLE_MR_KYBER */

// build the library with Naor Pinkas Base OT enabled
/* #undef ENABLE_NP */
```
So maybe this file is not used. (Debug is the building folder)

Any suggestions? Thanks!

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -165,10 +165,9 @@ if(ENABLE_SSE)
 endif()
 
 
-configure_file(libOTe/config.h.in libOTe/config.h)
-
+configure_file(libOTe/config.h.in ${CMAKE_CURRENT_SOURCE_DIR}/config.h)
 # Support out-of-source builds.
-include_directories(${CMAKE_BINARY_DIR})
+# include_directories(${CMAKE_BINARY_DIR})
 
 
 #############################################

